Gemfile.lock

```diff
@@ -0,0 +1,186 @@
+PATH
```

Comment:
We git-ignore Gemfile.lock to support multiple version of ruby & rails. Please remove this file

Reply:
done